routes/asset.js

```diff
@@ -70,25 +55,69 @@ router.get('/:id', (req, res) => {
   }
 });
 
-// // Route to Update asset name in database
-// router.get('/update/:id', (req, res) => {
-//   let newName = 'Updated_Asset_Name!';
-//   let sql = `UPDATE asset SET name = '${newName}' WHERE id = ${req.params.id}`;
-//   let query = connection.query(sql, (err, result) => {
-//     if (err) throw err;
-//     console.log(result);
-//     res.send(JSON.stringify(result));
-//   });
-// });
+// POST request to Create an asset
+router.post('/', (req, res) => {
+  let mysql = req.app.get('mysql');
+  let sql =
+    'INSERT INTO asset (name, physical_file_name, uploader, physical_file_size, physical_file_type) VALUES (?, ?, ?, ?, ?)';
+  let inserts = [
+    req.body.name,
+    req.body.fileName,
+    req.body.uploader,
+    req.body.fileSize,
+    req.body.fileType
+  ];
+  sql = mysql.pool.query(sql, inserts, (err, result) => {
+    if (err) {
+      res.write(JSON.stringify(err));
+      res.end();
+    } else {
+      res.redirect('/asset');
+    }
+  });
+});
 
-// //Delete asset from database
-// router.get('/delete/:id', (req, res) => {
-//   let sql = `DELETE FROM asset WHERE id = ${req.params.id}`;
-//   let query = connection.query(sql, (err, result) => {
-//     if (err) throw err;
-//     console.log(result);
-//     res.send(JSON.stringify(result));
-//   });
-// });
+// PUT request to Update asset in database
+router.put('/:id', function(req, res) {
+  let mysql = req.app.get('mysql');
+  // console.log(req.body)
+  // console.log(req.params.id)
+  let sql =
+    'UPDATE asset SET name=?, physical_file_name=?, uploader=?, physical_file_size=?, physical_file_type=? WHERE id=?';
+  let inserts = [
+    req.body.name,
+    req.body.fileName,
+    req.body.uploader,
+    req.body.fileSize,
+    req.body.fileType,
+    req.params.id
+  ];
+  sql = mysql.pool.query(sql, inserts, (err, results) => {
+    if (err) {
+      res.write(JSON.stringify(error));
+      res.end();
+    } else {
+      res.status(200);
+      res.end();
+    }
+  });
+});
+
+// DELETE request to Delete an asset from database
+router.delete('/:id', function(req, res) {
+  let mysql = req.app.get('mysql');
+  let sql = 'DELETE FROM asset WHERE id=?';
+  let inserts = [req.params.id];
+  sql = mysql.pool.query(sql, inserts, (err, results) => {
+    if (err) {
+      // console.log(error)
+      res.write(JSON.stringify(error));
```

Comment:
I don't see `error` defined anywhere. I think maybe you meant to do:
```suggestion
      res.write(JSON.stringify(err));
```
instead?